### DISCUSSION TITLE
[Q] 我设置列宽为10，但是excel文件确显示9.36

### DISCUSSION THREAD

**Discussion by Freedomhxb** [exceljs/exceljs#2366] [Q&A]
## 💬 Questions and Help
版本号4.3.0，
我设置列宽为10，但是单开excel文件确显示9.36，设置21显示20.36，这是为什么？哪位可以帮忙解决下，谢谢了。。
i set column with = 10, when i open xlsx file,it real is 9.36,who can help me? thanks

**Comment by zurmokeeper** [comment]
> ## 💬 Questions and Help
> 版本号4.3.0， 我设置列宽为10，但是单开excel文件确显示9.36，设置21显示20.36，这是为什么？哪位可以帮忙解决下，谢谢了。。 i set column with = 10, when i open xlsx file,it real is 9.36,who can help me? thanks

Any more code examples?
